docs/ml-features.md

```diff
@@ -53,7 +53,10 @@ collisions, where different raw features may become the same term after hashing.
 chance of collision, we can increase the target feature dimension, i.e. the number of buckets 
 of the hash table. Since a simple modulo is used to transform the hash function to a column index, 
 it is advisable to use a power of two as the feature dimension, otherwise the features will 
-not be mapped evenly to the columns. The default feature dimension is `$2^{18} = 262,144$`. 
+not be mapped evenly to the columns. The default feature dimension is `$2^{18} = 262,144$`.
+An optional binary toggle parameter controls term frequency counts. When set to true all nonzero frequency counts are
+set to 1. This is especially useful for discrete probabilistic models that model binary counts
+rather than integer.
```

Comment:
This sentence is not right.  The api doc reads "This is useful for discrete probabilistic models that model binary events rather than integer counts."